native_client/ctcdecode/scorer.cpp

```diff
@@ -83,7 +83,7 @@ void Scorer::setup(const std::string& lm_path, const std::string& trie_path)
   if (!has_trie) { // no trie was specified, build it now
     RetrieveStrEnumerateVocab enumerate;
     config.enumerate_vocab = &enumerate;
-    language_model_.reset(lm::ngram::LoadVirtual(filename, config));
+    language_model_.reset(new lm::ngram::QuantArrayTrieModel(filename, config));
```

Comment:
Why switch back to static model type? Was there significant overhead in `LoadVirtual`?

Reply:
Doc in the KenLM says we should avoid using virtual load at all expense.

Reply:
I know, but did it actually hurt performance? It's nice to be able to experiment with different data structures without having to change the code and recompile.

Reply:
I have not measured accurately, but I don't think it did. I'll recheck that.